include/LCUI/main.h

```diff
@@ -53,6 +53,7 @@ enum LCUI_SysEventType {
 	LCUI_PAINT,
 	LCUI_WIDGET,
 	LCUI_QUIT, /**< 在 LCUI 退出前触发的事件 */
+	LCUI_CARETPOSCHANGED,
```

Comment:
It is an input method related event, should be prefixed with IME, and CARETPOSCHANGED is not easy to read and should be changed to SETCARET, so you should rename it to `LCUI_IME_SETCARET`, just like `WM_SETCURSOR` and `WM_IME_SETCONTEXT`